pkg/cli/admin/upgrade/status/examples/4.16.0-ec2-control-plane-updated-pdb-prohibits-draining-node.yaml

```diff
@@ -1245,9 +1252,9 @@ items:
           type: PIDPressure
         - lastHeartbeatTime: "2024-02-27T16:19:15Z"
           lastTransitionTime: "2024-02-27T15:53:29Z"
-          message: kubelet is posting ready status
-          reason: KubeletReady
-          status: "True"
+          message: kubelet is still setting up stuff
+          reason: KubeletNotReady
+          status: "False"
```

Comment:
For my own reference, the reason we need to switch the `status` here is https://github.com/openshift/oc/blob/da9ac5f95c6103067385ec326b6c8b03463eb02a/pkg/cli/admin/upgrade/status/mco/layered_node_state.go#L251